modules/lib/global.d.ts

```diff
@@ -1,210 +1,112 @@
-interface XpLibraries {}
+import type {
+    App,
+    DoubleUnderscore,
+    Log,
+    NewBean as NewBeanType,
+    Resolve,
+    ScriptValue as ScriptValueInterface,
+    XpRequire,
+} from '@enonic-types/core';
+
+
+//──────────────────────────────────────────────────────────────────────────────
+// Declare global types
```

Comment:
Isn't it obvious what `declare global` is doing? Do we really need three-line comments here? We don't even separate several large code blocks in one large file with those comments.

Reply:
Removed